Initiating PR for new Azure AD Recommendations API (#19841)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/recommendation-dismiss.md

---
title: "recommendation: dismiss"
description: "Dismiss a recommendation object and updates its status to dismissed."
author: "hafowler"
ms.localizationpriority: medium
ms.prod: "identity-and-access"
doc_type: apiPageType
---

# recommendation: dismiss
Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Dismiss a [recommendation](../resources/recommendation.md) object that you consider to be inapplicable to your directory and update its **status** to `dismissed` .

Reviewer: ```suggestion
Dismiss a [recommendation](../resources/recommendation.md) object that you consider to be inapplicable to your tenant and update its **status** to `dismissed` .
```

## Permissions
One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

|Permission type|Permissions (from least to most privileged)|
|:---|:---|
|Delegated (work or school account)|DirectoryRecommendations.ReadWrite.All|
|Delegated (personal Microsoft account)|Not supported.|
|Application|DirectoryRecommendations.ReadWrite.All|

## HTTP request

<!-- {
  "blockType": "ignored"
}
-->
``` http
POST /directory/recommendations/{recommendationId}/dismiss
```

## Request headers
|Name|Description|
|:---|:---|
|Authorization|Bearer {token}. Required.|
|Content-Type|application/json. Required.|

## Request body
In the request body, supply a JSON representation of the parameters.

> The following table shows the parameters that can be used with this action.
> 
> |Parameter|Type|Description|
> |:---|:---|:---|
> |dismissReason|String|Explanation for why this recommendation is not applicable to your directory.|

Reviewer: ```suggestion
The following table shows the parameters that are required with this action.

|Parameter|Type|Description|
|:---|:---|:---|
|dismissReason|String|Explanation for why this recommendation is not applicable to your tenant.|
```
